packages/react-vega/package.json

```diff
@@ -18,11 +18,11 @@
     "types"
   ],
   "dependencies": {
-    "@types/react": "^16.9.6",
+    "@types/react": "^16.9.9",
     "vega-embed": "^6.0.0"
   },
   "peerDependencies": {
-    "react": "^16.10.2",
```

Comment:
We probably don't want to bump it too high if we don't need latest feature from `react`. This will show warnings in consumer projects if they use lower version of `react`.